tests/product/test_control.py

```diff
@@ -58,35 +58,35 @@ def assert_service_fails_without_presto(self, service):
         self.assertEqualIgnoringOrder(presto_not_installed,
                                       '\n'.join(start_output))
 
-    def test_server_start_various_states(self):
+    def test_server_start_coordinator_started(self):
         self.setup_cluster(NoHadoopBareImageProvider(),
                            self.STANDALONE_PRESTO_CLUSTER)
 
-        # Coordinator started, workers not; then server start
-        process_per_host = \
-            self.assert_start_with_one_host_started(
-                self.cluster.internal_master)
+        self.assert_start_with_one_host_started(self.cluster.internal_master)
```

Comment:
assert_start_with_one_host_started first starts that one host and then starts all of them.  Agree that we probably only need coordinator or workers and not both.  I'll remove one of the tests and change to test_server_start_one_host_started().